manifests/pipecd/values.yaml

```diff
@@ -187,6 +196,19 @@ prometheus:
               action: keep
               regex: admin
 
+    # TODO: Add Aleting rules
+    alerting_rules.yml:
+      groups:
+        - name: Instances
+          rules:
+            - alert: InstanceDown
+              expr: up == 0
+              for: 5m
+              labels:
+                severity: page
+              annotations:
+                description: '{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes.'
+                summary: 'Instance {{ $labels.instance }} down'
```

Comment:
Can this be loaded from external files like Grafana dashboards?

Reply:
I'll look up